README.md

```diff
@@ -2,7 +2,7 @@
 
 This is an extended list including a huge amount of products and services that we find for free!
 
-**Adding a service**
+--Adding a service--
```

Comment:
Please don't update this.

Reply:
Done.... I did it to **Adding a service**